### DISCUSSION TITLE
How remove a package?

### DISCUSSION THREAD

**Discussion by Olyno** [nachoaldamav/snpm#133] [Q&A] [answered]
Hi 👋🏻,
I'm not sure to understand how to remove (or add) a package with Ultra. Can someone explain me please?

**Comment by nachoaldamav** [comment, marked as answer]
Hi! To remove a package you can use `ultra remove package-name` and after that you should run `ultra install` again to create the node_modules again.
(It shows `Run "fnpm install" to update your node_modules` because the project was named like that before, i need to fix it)

![image](https://user-images.githubusercontent.com/22749943/197592945-ac1dea05-54c1-454d-9499-0fb76b91a2a4.png)

**Reply by Olyno** [reply]
Oh awesome! I thought there was no such thing. Would it be possible to update the documentation (CLI and [readme](https://github.com/nachoaldamav/ultra#-commands)) so that people who discover the project can find the command?

**Reply by nachoaldamav** [reply]
> Oh awesome! I thought there was no such thing. Would it be possible to update the documentation (CLI and [readme](https://github.com/nachoaldamav/ultra#-commands)) so that people who discover the project can find the command?

True, I'll add some commands to the README today!
